### PR TITLE
Migrate travis to github actions

### PR DESCRIPTION
**Pull request by alexander-schranz**
See CI in https://github.com/alexander-schranz/jackalope-jackrabbit/pull/1

### CODE REVIEW

File: .github/workflows/test-application.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+              uses: shivammathur/setup-php@v2
+              with:
+                  php-version: ${{ matrix.php-version }}
+                  tools: 'composer:v2'
+
+            - name: Install dependencies with Composer
+              uses: ramsey/composer-install@v1
+              with:
+                  dependency-versions: ${{ matrix.dependencies }}
+                  composer-options: --prefer-dist --no-suggest
```
Comment (dbu) [inline .github/workflows/test-application.yaml:47]:
```suggestion
                  composer-options: --prefer-dist
```

File: .github/workflows/test-application.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        strategy:
+            fail-fast: false
+
+            matrix:
+                php-version:
+                    - 7.3
+                    - 7.4
+                    - 8.0
+                dependencies:
+                    - highest
```
Comment (dbu) [inline .github/workflows/test-application.yaml:26]: i think this should be rewritten to the composer flag --prefer-lowest to be only used on the lowest php version instead.

**Review by dbu — COMMENTED** [review]
thanks. i think something with the matrix is odd here, and i think we lose the lowest version build.

can you please check the builds in your fork? it looks like the last version messed up something with jackrabbit - maybe it was a temporary hickup? as you seem to not have changed anything about how it gets installed...

File: .github/workflows/test-application.yaml (hunk 2 above)
Comment (alexander-schranz) [reply .github/workflows/test-application.yaml]: Yeah I think the combiniation between include and normal matrix is buggy in github action I moved all to the include now.

**Comment by dbu** [comment]
the last travis-ci build worked: https://travis-ci.com/github/jackalope/jackalope-jackrabbit/jobs/434093834

but that was 2 months ago. i now restarted it to see if the jackrabbit problem is with the changes here or with what we get when downloading

**Comment by dbu** [comment]
hm. still green. can you check how you install and start jackrabbit and how it was done before? looks like something has changed

**Comment by alexander-schranz** [comment]
@dbu downgraded ubuntu now its running [again](https://github.com/alexander-schranz/jackalope-jackrabbit/pull/1/checks). could not find out why it is not working with ubuntu 20 🙈

**Comment by dbu** [comment]
thanks! weird, no idea why that is. but as long as the tests run, i am happy. lets hope ubuntu 18 stays available for a while still...

**Pull request merged**